osu.Game/Screens/OnlinePlay/Multiplayer/Match/MultiplayerReadyButton.cs

```diff
@@ -36,17 +36,18 @@ protected override void LoadComplete()
         }
 
         private MultiplayerCountdown countdown;
-        private DateTimeOffset countdownReceivedTime;
+        private DateTimeOffset countdownChangeTime;
         private ScheduledDelegate countdownUpdateDelegate;
 
         private void onRoomUpdated() => Scheduler.AddOnce(() =>
         {
-            if (countdown == null && room?.Countdown != null)
-                countdownReceivedTime = DateTimeOffset.Now;
-
-            countdown = room?.Countdown;
+            if (countdown != room?.Countdown)
```

Comment:
Seems to work for now, but not 100% how well this reference comparison will hold up to future changes.